geops/geops-routing-demo
# feat(YAML test snippets): add debug mode with yaml test code generator popup

Create debug mode for showing YAML test snippets
Test:
- add 'mode=dev'
- Click 'bug'-Button in the menu next to the tabs to open the code snippet popup
- Add viaPoints
- Content in the code snippet should update and 5 expected viaPoints (crosses) should appear on the route
- Move the crosses around with the mouse cursor
- The corresponding coordinate should update in the code snippet
- Switch mot
- mot should update in the code snippet
- ...
